x/profiles/keeper/keeper_chain_links_test.go

```diff
@@ -355,6 +355,14 @@ func (suite *KeeperTestSuite) TestKeeper_DeleteChainLink() {
 					"cosmos",
 					ext.GetAddress().String(),
 				))
+
+				// Check the additional keys
+				store := suite.ctx.KVStore(suite.storeKey)
```

Comment:
We should have checking the additional keys in the store test function as well.

Reply:
Done